src/logger.ts

```diff
@@ -0,0 +1,5 @@
+import pino from 'pino';
+
+const logger = pino();
+
+export { logger };
```

Comment:
The justification for such a seemingly useless file here is if we wished to further configure the logger, that configuration would go here.

Reply:
100% -- this is a good solid file, the node way!!